### PR TITLE
Features

### PR DESCRIPTION
**Pull request by sergioobolevich**
In this project, we did a simple application in javascript using localStorage to save items on browser memory.
We added a toggle form that shows or hides the new book form.
We used a data- index property in the HTML element to be able to match it in the books array.
We added a read/unread and a delete button for each book.
A simple layout was created using flexbox, so the application is responsive design.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+buttonInput.addEventListener("click", createNewBook);
+
+function createNewBook() {
+  let title = bookTitleInput.value;
+  let desc = bookDescriptionInput.value;
+  let pages = pagesInput.value;
+  let status = readStatusInput.checked;
+
+  let book = {title, desc, pages, status};
+  // addBookToLibrary(book);
```
Comment (krys2fa) [inline app.js]: - [x] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). The same applies to line 160.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+buttonInput.addEventListener("click", createNewBook);
+
+function createNewBook() {
+  let title = bookTitleInput.value;
+  let desc = bookDescriptionInput.value;
+  let pages = pagesInput.value;
+  let status = readStatusInput.checked;
+
+  let book = {title, desc, pages, status};
```
Comment (krys2fa) [inline app.js]: - [x] [REQUIRED] Book should have a constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)).

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Great job @t-yanick and @chubaquelo  👏 
Your project is coming along nicely. 💪 
Just a few more adjustments to complete the project. They are seen below and in the comments.
### Status: Changes Required ♻️ 

- [x] [REQUIRED] The form should have an `author` field.
- [x] [OPTIONAL]  The form used to create a book should have validations, you should not be able to create an empty book which is the case now.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <button class="btn-add-new-book">Add New Book</button>
+    <div class="new-book">
+      <form>
+        <label for="title">Book Title:</label><br>
+        <input type="text" id="book-title" name="book-title"><br>
+        <label for="author">Book Author:</label><br>
+        <input type="text" id="book-author" name="book-author"><br>
+        <label for="pages">Number of pages:</label><br>
+        <input type="text" id="pages-number" name="pages-number"><br>
+        <input type="checkbox" id="read-status" name="read-status" value="Book">
```
Comment (SpaYco) [inline index.html:22]: - [ ] it would be better if the number of pages `<input>` is a `type` of `number`, not `text`, to reduce errors.

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**